### PR TITLE
#162979132 A user can view available meetups 

### PR DESCRIPTION
**Pull request by tirgei**
#### What does this PR do?
This Pull Request creates the main page where users can view meetups

#### Description of Task to be completed?
The tasks for this PR is:
- Create the main page 
- Style the main page
- Add the all meetups page
- Style the all meetups page

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/Questioner.git ```

CD into the project folder on your machine
```$ cd Questioner ```

Check out the branch for this PR
``` $ git checkout ft-user-meetups-162979132```

Open the `index.html` file on your browser

Alternatively, you can view the site here: [Questioner](https://tirgei.github.io/Questioner/UI)

#### Pivotal Tracker story associated with this PR
[#162979132](https://www.pivotaltracker.com/story/show/162979132)

#### Screenshots
index1

index2

index3

index4

meetups

Mobile 
mobile1

mobile2

mobile3

### CODE REVIEW

**Comment by wachiranduati** [comment]
Wow your designs are superb....your commit history super rich..nice one!!

**Comment by Ogollah** [comment]
This is Awesome.

**Comment by Ogollah** [comment]
One more thing commit is supposed to be in present. eg Add not Added.

File: UI/index.html [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+                    </li>
+                    <li>
+                        <a href="https://www.github.com/tirgei" class="github" target="_blank">
+                            <i class="fab fa-github"></i>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+    </footer>
+</html>
```
Comment (Ogollah) [inline UI/index.html:300]: new line here

File: UI/meetups.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                    </li>
+                    <li>
+                        <a href="https://www.github.com/tirgei" class="github">
+                            <i class="fab fa-github"></i>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+    </footer>
+</html>
```
Comment (Ogollah) [inline UI/meetups.html:194]: new line here

**Comment by luqee** [comment]
That is a nice clean interface.

**Pull request merged**